package.json

```diff
@@ -101,7 +101,13 @@
         "@semantic-release/release-notes-generator"
       ],
       "@semantic-release/npm",
-      "@semantic-release/github"
+      [
+        "@semantic-release/github",
+        {
+          "successComment": false,
+          "failTitle": false
+        }
+      ]
```

Comment:
This seems unrelated – what's the background here?

Reply:
Yeah, sorry I should have mentioned this on the description (which I will add). This is because of this: https://github.com/duffelhq/duffel-api-javascript/actions/runs/4881478277/jobs/8710431028

Reply:
Ah thanks for the context. I'm still a bit confused though – the thing you linked to mentions that we've hit a rate limiting issue?

Reply:
Yes, indeed. This is because every time we publish a new release we comment on GH issues which causes our workflow to fail. Please see https://github.com/semantic-release/semantic-release/issues/2204 for further informations on the work-around and issue

Reply:
Ah I see – thanks! It's a shame we can't add comments to package.json to give more context, but we can always check the git blame for it in the future and find this PR at least